soh/soh/Enhancements/randomizer/draw.cpp

```diff
@@ -12,7 +12,7 @@
 
 extern "C" void Randomizer_DrawSmallKey(PlayState* play, GetItemEntry* getItemEntry) {
     s32 pad;
-
+    s8 isColoredKeysEnabled = CVar_GetS32("gRandoMatchKeyColors", 0);
```

Comment:
i'm assuming this is because we used to always draw keysanity keys with colors, this makes it so it's a setting?

Reply:
Previously the check for using colored keys was done in the logic for deciding to use custom draw functions or not. This just moved the check inside the actual custom draw function so that logic could be cleaned up and be more flexible for colouring more items down the line.

See the changes in z_draw.c and the bottom of z_player_lib.c how it changed the logic there.